[MI-2608]: Explained the jira subscription modal in the doc. (#31)
Brightscout/mattermost-plugin-jira: docs/administrator-guide/notification-management.md

![This is the Channel Subscription modal](../.gitbook/assets/channel-subscriptions-modal.png)

## How to create and use the subscription modal?

1. Type the `/jira subscribe` command to create the subscription in the particular channel.
2. Click on the **Create Subscription** button to receive the Jira issue notifications.

> 3. Write the name of a subscription in the **Subscription Name** field.

Reviewer: ```suggestion
3. Write the name of the subscription in the **Subscription Name** field.
```

4. Select the project name to which you want to subscribe.

> 5. Select the events and issue type for which you want to receive the notification in the mattermost.

Reviewer: ```suggestion
5. Select the events and issue type for which you want to receive the notification in the mattermost channel.
```

> 6. To be more specific you can add filters as well.You can include or exclude the particular filter by mentioning its type or name. For now, we have 3 types of filters :

Reviewer: ```suggestion
6. To be more specific you can add filters as well. You can either include or exclude the particular filter by mentioning its type or name. For now, we have 3 types of filters :
```

   * Epic Link
   * Labels
   * Priority

> 7. Click on the **Add Subscription** button to add the subscription to that channel with the specific constraints that you have selected above.

Reviewer: Include some insights about the JQL query as well

![This is the Channel Subscription modal](../.gitbook/assets/channel-subscriptions-modal.png)

## How to create and use the subscription modal?

> 1. Type the `/jira subscribe` command to create the subscription in the particular channel.

Reviewer 2: Please add a recorded video of how this modal works in the PR description, so that I can verify the written steps with the video.

Author: [JiraSubscriptionModalWorking.webm](https://user-images.githubusercontent.com/107465508/213691139-75b3fe13-d4f6-43de-a85d-280a35e74328.webm)

Reviewer 2: Please add this video in the PR description.

Reviewer 2: ```suggestion
1. Type the `/jira subscribe` command to open the "Create subscription" modal in the particular channel.
```

2. Click on the **Create Subscription** button to create the subscription and receive the Jira issue notifications.
3. Write the name of the subscription in the **Subscription Name** field.
4. Select the project name to which you want to subscribe.

> 5. Select the events and issue type for which you want to receive the notification in the mattermost channel.

Reviewer 2: ```suggestion
5. Select the events and issue types for which you want to receive the notifications in the Mattermost channel.
```

> 6. To be more specific you can add filters as well. You can either include or exclude the particular filter by mentioning its type or name. For now, we have 3 types of filters :

Reviewer 2: ```suggestion
6. To be more specific, you can add filters as well. You can either include or exclude the particular filter by adding its type or name. For now, we have 3 types of filters :
```
